### PR TITLE
Updated documentation for Conan Adapter

### PR DESCRIPTION
**Pull request by ChGen**
Part of #5 - provides FullIndexer class with tests for Artipie SDK.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,38 @@ This is the dependency you need:
 </dependency>
 ```
 
+After that, you can use existing `com.artipie.asto.Storage` implementations to create required storage object. 
+There are filesystem storage, S3 storage, in-memory storage types.
+
+Then, you make an instance of `Rpm` class with your storage
+as an argument. Finally, you put your artifacts to the storage specifying repository key 
+(`rpm-repo` in our example) and instruct `Rpm` to update the meta info:
```
Comment (g4s8) [inline README.md:118]: @ChGen why RPM here?

File: README.md [hunk 2]
```diff
@@ -110,6 +110,38 @@ This is the dependency you need:
 </dependency>
 ```
 
+After that, you can use existing `com.artipie.asto.Storage` implementations to create required storage object.
```
Comment (g4s8) [inline README.md:113]: It'd be good to add a link to asto repo documentation

File: README.md [hunk 3]
```diff
@@ -110,6 +110,38 @@ This is the dependency you need:
...
+(`rpm-repo` in our example) and instruct `Rpm` to update the meta info:
+
+# RevisionsIndexApi
+
+This class provides APIs for Conan revision index support for Conan V2 protocol.
+It provides methods to add and remove revision information from revisions.txt files, for example:
+CompletionStage<Void> addRecipeRevision(final int revision);
+CompletionStage<Boolean> removeRecipeRevision(final int revision);
+CompletionStage<List<Integer>> getRecipeRevisions();
+CompletionStage<List<Integer>> getBinaryRevisions(final int reciperev, final String hash);
```
Comment (g4s8) [inline README.md:127]: This should be wrapped with markdowns code format symbols (tripple `)

File: README.md [hunk 4]
```diff
@@ -110,6 +110,38 @@ This is the dependency you need:
...
+
+It also provides methods for index files update after package contents changes:
+CompletionStage<List<Integer>> updateRecipeIndex();
+CompletionStage<List<Integer>> updateBinaryIndex(final int reciperev, final String hash);
+CompletionStage<Void> fullIndexUpdate()
```
Comment (g4s8) [inline README.md:132]: And same here (formatting)

File: README.md [hunk 5]
```diff
@@ -110,6 +110,38 @@ This is the dependency you need:
...
+Usage example:
+
+```java
+package com.artipie.conan;
+final Storage storage = new InMemoryStorage();
+final RevisionsIndexApi index = new RevisionsIndexApi(storage, new Key.From("zlib/1.2.11/_/_"));
+index.fullIndexUpdate().toCompletableFuture().join();
+```
+
+See also Artipie Conan adapter javadoc.
```
Comment (g4s8) [inline README.md:143]: You can add a link to auto-generated HTML javadocs here (see javadoc badges in other repos to get thelink)

**Review by g4s8 — CHANGES_REQUESTED** [review]
@ChGen please check my comments

File: README.md (hunk 1 above)
Comment (ChGen) [reply README.md:118]: @g4s8 , oh, sorry, those training courses made me inattentive, I'll finish with them first :/

**Comment by ChGen** [comment]
@g4s8 , please review. I added two links also.

**Review by g4s8 — APPROVED** [review]
@ChGen thanks

**Pull request merged**